VirtoCommerce.Platform.Web/Controllers/Api/SettingController.cs

```diff
@@ -94,7 +93,21 @@ public async Task<ActionResult> UpdateAsync([FromBody] ObjectSettingEntry[] obje
         [ApiExplorerSettings(IgnoreApi = true)]
         public async Task<ActionResult<object[]>> GetArrayAsync(string name)
         {
-            var result = (await _settingsManager.GetObjectSettingAsync(name)).AllowedValues;
+            var settings = await _settingsManager.GetObjectSettingAsync(name);
+            object[] result;
```

Comment:
Make this line a first line in the method and separate it with an empty line from the remaining code.

Reply:
fixed